parent/pom.xml

```diff
@@ -260,7 +260,7 @@
         <jedis-client-version>5.1.0</jedis-client-version>
         <jetcd-version>0.7.7</jetcd-version>
         <jetty-version>12.0.5</jetty-version>
-        <jetty-plugin-version>${jetty-version}</jetty-plugin-version>
+        <jetty-maven-plugin-version>11.0.19</jetty-maven-plugin-version>
```

Comment:
Is there not a v12 of the jetty maven plugin ?

Reply:
No, the Maven plugin version does not track with the Jetty version. Highest plugin version is 11.0.20, released 31 Jan, 2024.